platform/authorized_api/users/index.json

```diff
@@ -19,14 +19,16 @@
   "definitions": {
     "User_Role": {
       "type": "string",
+      "description": "User role",
```

Comment:
Do you want to add missing descriptions to previous versions?

Reply:
imo good point, @plks will you prepare a PR?

Reply:
If you want to do that - it'll be better to wait with merge of this branch until these changes will be on target branch. I'll let us rebase this branch against it and remove duplication of description changes in this PR.